app/bt/nativeModule.ts

```diff
@@ -72,12 +73,66 @@ export const getCurrentENPermissionsStatus = async (
 };
 
 // Key Submission Module
+const keySubmissionModule = NativeModules.KeySubmissionModule;
+
 export const submitDiagnosisKeys = async (
   cb: (errorMessage: string, successMessage: string) => void,
 ): Promise<void> => {
   keySubmissionModule.postDiagnosisKeys(cb);
 };
 
+// Exposure Key Module
+// const exposureKeyModule = NativeModules.ExposureKeyModule;
+export const getExposureKeys = async (): Promise<ExposureKey[]> => {
```

Comment:
This is outstanding work to wire up the native layer to handle the promise instead of the call back.